src/composables/useMessageInfo.js

```diff
@@ -61,6 +61,8 @@ export function useMessageInfo(message = ref({})) {
 	const isBotInOneToOne = computed(() =>
 		message.value.actorId.startsWith(ATTENDEE.BOT_PREFIX)
 		&& message.value.actorType === ATTENDEE.ACTOR_TYPE.BOTS
+		&& (conversation.value.type === CONVERSATION.TYPE.ONE_TO_ONE
+			|| conversation.value.type === CONVERSATION.TYPE.ONE_TO_ONE_FORMER)
```

Comment:
Would be better to maintain
```suggestion
		&& isOneToOneConversation.value
```

Reply:
I didn't find it